BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -439,34 +374,28 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
                 }
             });
         }];
-        return NO;
-    } else if (![[[request URL] absoluteString] isEqualToString:[[request mainDocumentURL] absoluteString]]) {
-        // If it is an iFrame, there's not much we can do. We have to just let the UIWebView do the load.
-        // If we tried to use NSURLSession to make this request, we would not know how to properly populate the
-        // iframe with the response.
-        // This means we cannot handle scenarios such as:
-        // a) The iFrame request requires HTTP Auth. We would normally want to pop up a custom dialog to collect credentials.
-        // b) The iFrame request has an invalid SSL certificate. We would normally want to pop up a warning dialog and let the user decide what to do.
-        return YES;
-    } else if ([self webViewCanBeUsedDirectlyForHost:request.URL.host] == NO) {
-        BOXLog(@"Was not authenticated, launching URLSession and not loading the request in the web view");
-        [self.activityIndicator startAnimating];
-        NSURLSessionDataTask *dataTask = [self.URLSession dataTaskWithRequest:request];
-        [dataTask resume];
-        BOXLog(@"URLSessionDataTask is %@", dataTask);
-        return NO;
+        decisionHandler(WKNavigationActionPolicyCancel);
+        return;
+    }
+    if (@available(iOS 11.0, *)) {
```

Comment:
Does this `if` need and `else`?

Reply:
I think the else is just to fall through.